documentation/api-scenario/how-to/IntegrateWithTestProxy.md

```diff
@@ -0,0 +1,87 @@
+# How to integrate test-proxy with API Scenario test
```

Comment:
The title is a little bit confusing.  in fact, to run API scenario test does not require to 'integrate' test-proxy.  to install test proxy is to record traffic for traffic validation and SDK recording test.  However, the title makes me think API scenario test requires to integrate test-proxy.